tests/cases/compiler/varianceCantBeStrictWhileStructureIsnt.ts

```diff
@@ -0,0 +1,28 @@
+// @strict: false
+// @strictFunctionTypes: false

```

Comment:
Maybe it'd be better to do `false, true` for these?

Reply:
We already have tests covering the `true` case (as part of much larger tests - basically any variance-related test), just not the `false`.